runtime/gear/src/lib.rs

```diff
@@ -98,6 +98,10 @@ mod migrations;
 // Weights used in the runtime.
 mod weights;
 
+#[link_section = "wasm_runtime_version"]
+static _RUNTIME_VERSION: [u8; const_str::to_byte_array!(env!("SUBSTRATE_CLI_IMPL_VERSION")).len()] =
```

Comment:
u can just &str here and it will be utf-8 encoded as bytes in blob, isn't it?

Reply:
Using references with named sections doesn't work with the wasm target. It requires it to be a simple fixed size array. I would not do all that trible dancing on my own.